service/src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -782,4 +782,19 @@ public List<WorkspaceUserPair> getPrivateResourceUsers() {
 
     return jdbcTemplate.query(sql, params, WORKSPACE_USER_PAIR_ROW_MAPPER);
   }
+
+  /**
+   * Temporary backfill of cloud context spend profile from workspace spend profile TODO: PF-2762
+   * remove this backfill.
+   */
+  @WriteTransaction
+  public void backfillCloudContextSpendProfile() {
```

Comment:
fwiw I think you could also do this in another liquibase changeset, but either way seems fine